src/index.html

```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <div class="stars-star"></div>
```

Comment:
There is BEM methodology, so use `stars__star` instead
Make changes all over the document